cli/cmd/cluster.go

```diff
@@ -212,10 +232,16 @@ func (c controlplaneUpdater) upgradeComponent(component string) {
 
 		install := action.NewInstall(actionConfig)
 		install.ReleaseName = component
-		install.Namespace = "kube-system"
+		install.Namespace = namespace
 		install.Atomic = true
 
-		if _, err := install.Run(helmChart, map[string]interface{}{}); err != nil {
```

Comment:
Hmm, this seems like a bug, that `values` are not used here. Can you verify that by uninstalling some controlplane component (e.g. calico) and then running the apply? I'd expect it to be configured wrong. If that's the case, please fix that in separate commit.